### PR TITLE
Add some logs in DTLSConnector about sending ALERT & refuse RENEGOCIAION

### PR DESCRIPTION
**Pull request by sbernard31**
This was triggered by https://github.com/eclipse/leshan/issues/754.

### CODE REVIEW

File: scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java [hunk 1]
```diff
@@ -1391,9 +1391,12 @@ private void processHandshakeRecord(final Record record, final Connection connec
 				// We do not support re-negotiation as recommended in :
 				// https://tools.ietf.org/html/rfc7925#section-17
 				if (record.getEpoch() > 0) {
```
Comment (sbernard31) [inline scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java]: I asked my self if this `if (record.getEpoch() > 0) {` is really needed as some line above we check that this is not a new CLIENT_HELLO ?

Comment (boaks) [reply scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java]: > as some line above we check that this is not a new CLIENT_HELLO ?

do you mean 

```
public boolean isNewClientHello() {
	if (0 < epoch || type != ContentType.HANDSHAKE || 0 == fragmentBytes.length) {
		return false;
	}
	HandshakeType handshakeType = HandshakeType.getTypeByCode(fragmentBytes[0]);
	return handshakeType == HandshakeType.CLIENT_HELLO;
}
```

that is only for epoch 0 CLIENT_HELLOs. So my feeling is, this check seems to be not too bad.

Comment (boaks) [reply scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java]: OK, I got it. Yes, it look like it' obsolete in the meantime.

**Comment by boaks** [comment]
LGTM1
Do you want to remove the `if` and then merge it before 2.0.0-M18?

**Comment by sbernard31** [comment]
> Do you want to remove the if and then merge it before 2.0.0-M18?

Yep

**Comment by sbernard31** [comment]
`if` is removed.

Do I merge it now ? or I let you merge this ?

**Pull request merged**